src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,181 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import file.FileManager;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static final String horizLine = "____________________________________________________________";
+
+    public static void main(String[] args) throws IOException {
+        ArrayList<Task> tasks = new ArrayList<>();
+        showWelcomeMessage();
+        FileManager.loadFile(tasks);
+        executeUserCommand(tasks);
+    }
+
+    private static void executeUserCommand(ArrayList<Task> tasks) throws IOException {
+        String command;
+        Scanner in = new Scanner(System.in);
+        command = in.nextLine();
+
+        while (!command.toLowerCase().equals("bye")) {
```

Comment:
Noticed that there are repeated `command.toLowerCase()`, perhaps you can declare a new `String `with a lowered case variable?